quickstarts/calico/config.yml

```diff
@@ -24,18 +24,22 @@ keywords:
   - k8s
   - prometheus
   - remote-write
+  - cni
 
 installPlans:
   - setup-prometheus
+  - setup-prometheus-agent-mode
```

Comment:
This install plan does not exist. Please use an existing one or create your own!

Reply:
pending in https://github.com/newrelic/newrelic-quickstarts/pull/1503